crawler/config/centres.py

```diff
@@ -239,3 +263,42 @@
         CENTRE_KEY_SKIP_UNCONSOLIDATED_SURVEILLANCE_FILES: False,
     },
 ]
+
+
+def get_centres_config(config: Config, data_source: str = "") -> List[CentreConf]:
+    """Get the centres config from MongoDB. If MongoDB does not contain any centres config, it will become populated
+    with the values in the app config for centres.
+
+    Arguments:
+        config {Config}: The configuration object for the whole application.
+        data_source {str}: The data source filter to apply to centre configs, or None to apply no filter.
+
+    Return:
+        List[CentreConf]: A List of CentreConf from MongoDB matching the given data source.
+    """
+    with create_mongo_client(config) as client:
+        db = get_mongo_db(config, client)
+
+        centres_collection_exists = collection_exists(db, COLLECTION_CENTRES)
+        centres_collection = get_mongo_collection(db, COLLECTION_CENTRES)
```

Comment:
why get the centres collection here? if in line 282 you find the collection doesn't exist, it won't be there, and you then create it below here in lines starting 285.
I may be misunderstanding what's happening here..

Reply:
Right yes, but I need a reference to it for line 287 and 288 inside the `if` block, and then also outside the `if` block again on line 291.  PyMongo creates the collection with this getter if it doesn't exist.